client/src/pages/application/student/student_application.js

```diff
@@ -225,6 +225,25 @@ const StudentApplication = () => {
         </Alert>
       )}
       <h3>Availability</h3>
+      {termApp.hasOwnProperty("funding") && (
+        <Alert severity="info">
+          <b>
+            Load required for funding: {termApp?.funding} quarter load
+            {termApp?.funding === 1 ? "" : "s"}
```

Comment:
That `s` is supposed to handle plural...

```suggestion
            Load required for funding: {termApp?.funding} quarter load{termApp?.funding === 1 ? "" : "s"}
```

Reply:
Bumping the message above to make sure you saw it since you added a comment after this... @kukuminer

Reply:
Yes, it's there. It is in the PR. It works as expected. The linter just put it on the following line, but it works as you would expect. 1 quarter load, 0/2/3/4 quarter load*s*

Reply:
[Interesting](https://stackoverflow.com/a/38547635)... Anyway, approval stands.